Composer/packages/client/src/pages/design/DesignPage.tsx

```diff
@@ -708,6 +810,37 @@ const DesignPage: React.FC<RouteComponentProps<{ dialogId: string; projectId: st
             onDismiss={() => dismissManifestModal(projectId)}
           />
         )}
+        {brokenSkillInfo && (
+          <RepairSkillModal
+            skillItem={brokenSkillInfo}
+            onDismiss={() => {
+              setBrokenSkillInfo(undefined);
+            }}
+            onNext={(option) => {
+              const skillIdToRemove = brokenSkillInfo.skillId;
+              if (!skillIdToRemove) return;
+
+              if (option === RepairSkillModalOptionKeys.repairSkill) {
+                setCreationFlowTypes('Skill');
+                setCreationFlowStatus(CreationFlowStatus.OPEN);
+                setBrokenSkillRepairCallback(() => {
+                  removeSkillFromBotProject(skillIdToRemove);
+                });
+              } else if (option === RepairSkillModalOptionKeys.removeSkill) {
+                removeSkillFromBotProject(skillIdToRemove);
+              }
+              setBrokenSkillInfo(undefined);
+            }}
+          ></RepairSkillModal>
+        )}
+        <CreationModal
+          onDismiss={() => {}}
```

Comment:
is there a reason why this is not implemeneted. If not this prop should be made optional inside CreationModal